sshd-entrypoint.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+# Alternative entrypoint for the inmanta orchestrator container.
+# Instead of starting the orchestrator, it starts an ssh daemon, to allow
+# remote access to some volumes shared with the actual orchestrator.
+
+set -x
+set -e
+
+## INPUTS ##
+# Get the public key which should be inserted into the authorized keys file of
+# the inmanta user
+INMANTA_USER_AUTHORIZED_KEYS="${INMANTA_AUTHORIZED_KEYS}"
+
+
+## SETUP ##
+# Configure inmanta cli, persist any of the provided environment variables
+# in the user profile file
+INMANTA_USER_HOME_DIR=$(getent passwd inmanta | cut -d: -f6)
+export | grep INMANTA >> "${INMANTA_USER_HOME_DIR}/.profile"
```

Comment:
This I don't understand

Reply:
export will show all the environment variables available in the container, in the form `declare -x ...`, we pipe these to the profile file of the inmanta user so it can also use the environment variables configured on the container.  These environment variables are required to setup the cmdline_rest_transport options for example.

Reply:
But how do they get into the container at all?

Reply:
With the `environment` section in the docker compose file